From 76588a4001b0527d1cf04b15bfcd70da72d06b93 Mon Sep 17 00:00:00 2001
From: Bobby The Builder <bob@najdan.com>
Date: Thu, 29 Jul 2021 07:43:59 -0400
Subject: [PATCH 2/2] bypass concatenation issue during build

   ld: unrecognized option '--sort-common;-Wl'
   [CMakeFiles/olm.dir/build.make:434: libolm.so.3.2.4] Error 1
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0bd5ec8..bfe990a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,7 +49,7 @@ add_library(olm
 add_library(Olm::Olm ALIAS olm)
 
 # restrict the exported symbols
-set(CMAKE_SHARED_LINKER_FLAGS ${CMAKE_SHARED_LINKER_FLAGS} -Wl,--version-script=${CMAKE_CURRENT_SOURCE_DIR}/libolm.version)
+set(CMAKE_SHARED_LINKER_FLAGS -Wl,--version-script=${CMAKE_CURRENT_SOURCE_DIR}/libolm.version)
 set_target_properties(olm PROPERTIES LINK_DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/libolm.version)
 
 target_include_directories(olm
-- 
2.31.1

